cis_v120/section_1.pp

```diff
@@ -0,0 +1,159 @@
+locals {
+  cis_v120_1_common_tags = merge(local.cis_v120_common_tags, {
+    cis_section_id = "1"
+  })
+}
+
+locals {
+  cis_v120_1_1_common_tags = merge(local.cis_v120_1_common_tags, {
+    cis_section_id = "1.1"
+  })
+}
+
+benchmark "cis_v120_1" {
+  title         = "1 Directory"
+  documentation = file("./cis_v120/docs/cis_v120_1.md")
+  children = [
+    benchmark.cis_v120_1_1
+  ]
+
+  tags = merge(local.cis_v120_1_common_tags, {
+    type    = "Benchmark"
+    service = "GoogleWorkspace/Directory"
+  })
+}
+
+benchmark "cis_v120_1_1" {
+  title         = "1.1 Users"
+  documentation = file("./cis_v120/docs/cis_v120_1_1.md")
+  children = [
+    control.cis_v120_1_1_1,
+    control.cis_v120_1_1_2,
+    control.cis_v120_1_1_3
+  ]
+
+  tags = merge(local.cis_v120_1_1_common_tags, {
+    type    = "Benchmark"
+    service = "GoogleWorkspace/Directory"
+  })
+}
+
+control "cis_v120_1_1_1" {
+  title         = "1.1.1 (L1) Ensure more than one Super Admin account exists"
+  description   = "Having more than one Super Admin account is needed primarily so that a single point of failure can be avoided."
+  sql           = query.admin_super_admin_count.sql
+  documentation = file("./cis_v120/docs/cis_v120_1_1_1.md")
+
+  tags = merge(local.cis_v120_1_1_common_tags, {
+    cis_item_id = "1.1.1"
+    cis_level   = "1"
+    cis_type    = "manual"
+    service     = "GoogleWorkspace/Directory"
+  })
+}
+
+control "cis_v120_1_1_2" {
+  title         = "1.1.2 (L1) Ensure no more than 4 Super Admin accounts exist"
+  description   = "Having more than one Super Admin account is needed primarily so that a single point of failure can be avoided, but having too many should be avoided."
+  sql           = query.admin_super_admin_max_count.sql
```

Comment:
@Priyanka-Chatterjee-2000 please update the `sql` references to `query` throughout this benchmark.

Reference: https://github.com/turbot/steampipe-mod-aws-compliance/blob/main/cis_v500/section_2.pp#L53